### PR TITLE
docs: Move derivation tutorial to Tutorial section

### PR DESCRIPTION
**Pull request by danthelion**
**Description:**

The AcmeBank tutorial is a nice lengthy guide on how to set up Derivations and it currently lives in the Concepts/Derivations page. It would be great if we could keep the docs consistent and have all "learn-by-doing" style tutorials in the Tutorials section, so the "Concepts" page can stay lean and focus on the concepts.

**Documentation links affected:**

- [AcmeBank tutorial](https://docs.estuary.dev/concepts/derivations/#introducing-acmebank)

- - -
This change is [Reviewable](https://reviewable.io/reviews/estuary/flow/1462)

### CODE REVIEW

**Comment by jgraettinger** [comment]
@jonwihl  ping

File: site/docs/concepts/derivations.md [hunk 1]
```diff
@@ -30,307 +29,6 @@ Today, Flow enables you to write derivations using either
 [SQLite](#sqlite) or [TypeScript](#typescript).
 Additional language support is in the works.
```
Comment (jgraettinger) [inline site/docs/concepts/derivations.md:31]: What about a visible link here for " if you'd like a tutorial introduction to derivations, see ..." etc to let users self-select whether they want this reference-y document or the acmeBank examples.

Comment (danthelion) [reply site/docs/concepts/derivations.md:31]: Good idea, added - thanks!

**Review by jgraettinger — APPROVED** [review]
LGTM

**Pull request merged**